lib/linguist/languages.yml

```diff
@@ -4884,6 +4880,14 @@ Objective-J:
   tm_scope: source.js.objj
   ace_mode: text
   language_id: 259
+Octave:
+  type: programming
+  color: "#2299c4"
+  extensions:
+  - ".m"
+  tm_scope: source.octave
+  ace_mode: octave
+  language_id: 312
```

Comment:
Please follow the CONTRIBUTING.md details for adding a new language and generating this ID.

Reply:
A proper language_id has been generated and added to the `languages.yml` file.